src/controllers/auth.ts

```diff
@@ -62,4 +62,18 @@ export const loginUser = (req: Request, res: Response) => {
             return res.status(501).json({ message: "internal server error" })
         }
         );
-}
\ No newline at end of file
+}
+
+export const authenticateToken = (req: any, res: Response, next: NextFunction) => {
```

Comment:
Please define a propper type for the request var, `any` is only okay while developing locally.